Nelsonomotuyole.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Nelson omotuyole",
+  "displapstatsPublic": true,
+  "type": "personal",
+  "bio": "Teacher turned **programmer** | Intergrated science student from *The university of lagos Nigeria | presently Learning **phyton pro bootcamp with angela yu**  ",
```

Comment:
one more typo I found here, it should be `python` instead of `phyton`

Reply:
okay thanks sir